### PR TITLE
Updated README to add source acquisition instructions  in 'Building' section

### PR DESCRIPTION
**Pull request by jouyouyun**
Sync from `Building from Git` section.

### CODE REVIEW

**Comment by LDVG** [comment]
Hi,

Thank you for your contribution. 

These dependencies are not included in the `Building` section because it's assumed that the build happens out of one of our releases [^1] and these tarballs already include pre-built manual pages. I'd argue that the dependency list then is correct for that section, is there something else we could clarify in this matter?

[^1]: https://developers.yubico.com/pam-u2f/Releases/

**Comment by jouyouyun** [comment]
Seeing your reply, I understand. However, it is recommended to explain the download method in the Building section to avoid misunderstandings.

File: README [hunk 1]
```diff
@@ -11,6 +11,9 @@ infrastructure.
 [[building]]
 == Building
 
+You may get the source code from the
+https://github.com/Yubico/pam-u2f/tags[Release Tags].
```
Comment (LDVG) [inline README:15]: The signed tarballs (which include the pre-built manual pages) are only available through Yubico's website. What do you think of the following suggestion?

```suggestion
Signed release tarballs are available at Yubico's
https://developers.yubico.com/pam-u2f/Releases['release page'].
```

**Review by LDVG — CHANGES_REQUESTED** [review]

File: README (hunk 1 above)
Comment (jouyouyun) [reply README]: ok

**Review by LDVG — APPROVED** [review]
Thank you!

**Pull request merged**